opm/simulators/wells/WellState.hpp

```diff
@@ -267,7 +267,8 @@ namespace Opm
                 //    the well is an injector or producer)
                 //    pressure in first perforation cell.
                 const int first_cell = well_perf_data_[w][0].cell_index;
-                const double safety_factor = well.isInjector() ? 1.01 : 0.99;
+                // const double safety_factor = well.isInjector() ? 1.01 : 0.99;
+                const double safety_factor = well.isInjector() ? 1.05 : 0.90;
```

Comment:
this is to make sure the perforations are producing for producers at the beginning of the iteration.  It might not be necessary.

Reply:
It is reverted because it is found not essential. And also it can avoid failing tests.